hyperspy/tests/model/test_components2D.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+import nose.tools as nt
+
+import hyperspy.api as hs
+
+
+class TestGaussian2D:
```

Comment:
Thanks for that - I'll move this to test_model and add a new test that does what you describe